### DISCUSSION TITLE
how to disable the bold font in the af-magic theme

### DISCUSSION THREAD

**Discussion by bigben446** [ohmyzsh/ohmyzsh#10842] [Help] [answered]
when I ls the folder, the subfolders are bold fonts. I need bold fonts globally disabled.
https://storage.deepin.org/thread/202204111500061485_%E6%88%AA%E5%9B%BE_%E9%80%89%E6%8B%A9%E5%8C%BA%E5%9F%9F_20220411145414.jpg

A similar discussion is here:

> Add option to print prompt in non-bold font #426
> https://github.com/spaceship-prompt/spaceship-prompt/issues/426
> 
> Issue
> I'd love for Spaceship to have option to not set bold as the default font style for the prompt. What do people think about this? If others are interested in this as well, I'd start prototyping a PR.
> 
> Screenshot
> iTerm with bold fonts globally disabled:
> https://user-images.githubusercontent.com/406937/39147121-10d0486c-4739-11e8-958f-4cc7fcfe8ca2.png
> Hyper with the Spaceship default:
> https://user-images.githubusercontent.com/406937/39147118-0f577cd0-4739-11e8-8e4a-47de7a868de3.png

similar discussion:
> Can I remove bold text from zsh
> https://unix.stackexchange.com/questions/457855/can-i-remove-bold-text-from-zsh

**Comment by mcornella** [comment, marked as answer]
The `af-magic` theme does not use any bold fonts. If what you mean is how to remove bold output for the `ls` command, you need to modify either [`$LSCOLORS` or `$LS_COLORS`](https://gist.github.com/thomd/7667642) depending on your `ls` version.

Post the output of `ls --version` and `typeset -p LSCOLORS LS_COLORS` so I can guide you towards removing bold formatting for `ls`.

The similar issues you posted also mention globally-disabling bold text. I assume that's not what you need and you only need to disable bold formatting in `ls` output.

**Reply by bigben446** [reply]
thank you very much!

ls --version
ls (GNU coreutils) 8.30

typeset -p LSCOLORS LS_COLORS                                                                                        
export LSCOLORS=Gxfxcxdxbxegedabagacad
export LS_COLORS='rs=0:di=01;34:ln=01;36:mh=00:pi=40;33:so=01;35:do=01;35:bd=40;33;01:cd=40;33;01:or=40;31;01:mi=00:su=37;41:sg=30;43:ca=30;41:tw=30;42:ow=34;42:st=37;44:ex=01;32:*.tar=01;31:*.tgz=01;31:*.arc=01;31:*.arj=01;31:*.taz=01;31:*.lha=01;31:*.lz4=01;31:*.lzh=01;31:*.lzma=01;31:*.tlz=01;31:*.txz=01;31:*.tzo=01;31:*.t7z=01;31:*.zip=01;31:*.z=01;31:*.dz=01;31:*.gz=01;31:*.lrz=01;31:*.lz=01;31:*.lzo=01;31:*.xz=01;31:*.zst=01;31:*.tzst=01;31:*.bz2=01;31:*.bz=01;31:*.tbz=01;31:*.tbz2=01;31:*.tz=01;31:*.deb=01;31:*.rpm=01;31:*.jar=01;31:*.war=01;31:*.ear=01;31:*.sar=01;31:*.rar=01;31:*.alz=01;31:*.ace=01;31:*.zoo=01;31:*.cpio=01;31:*.7z=01;31:*.rz=01;31:*.cab=01;31:*.wim=01;31:*.swm=01;31:*.dwm=01;31:*.esd=01;31:*.jpg=01;35:*.jpeg=01;35:*.mjpg=01;35:*.mjpeg=01;35:*.gif=01;35:*.bmp=01;35:*.pbm=01;35:*.pgm=01;35:*.ppm=01;35:*.tga=01;35:*.xbm=01;35:*.xpm=01;35:*.tif=01;35:*.tiff=01;35:*.png=01;35:*.svg=01;35:*.svgz=01;35:*.mng=01;35:*.pcx=01;35:*.mov=01;35:*.mpg=01;35:*.mpeg=01;35:*.m2v=01;35:*.mkv=01;35:*.webm=01;35:*.ogm=01;35:*.mp4=01;35:*.m4v=01;35:*.mp4v=01;35:*.vob=01;35:*.qt=01;35:*.nuv=01;35:*.wmv=01;35:*.asf=01;35:*.rm=01;35:*.rmvb=01;35:*.flc=01;35:*.avi=01;35:*.fli=01;35:*.flv=01;35:*.gl=01;35:*.dl=01;35:*.xcf=01;35:*.xwd=01;35:*.yuv=01;35:*.cgm=01;35:*.emf=01;35:*.ogv=01;35:*.ogx=01;35:*.aac=00;36:*.au=00;36:*.flac=00;36:*.m4a=00;36:*.mid=00;36:*.midi=00;36:*.mka=00;36:*.mp3=00;36:*.mpc=00;36:*.ogg=00;36:*.ra=00;36:*.wav=00;36:*.oga=00;36:*.opus=00;36:*.spx=00;36:*.xspf=00;36:'

**Reply by bigben446** [reply]
export LS_COLORS=${LS_COLORS}'di=00;34:ln=00;36'

solve the problem, thank you very much!
